pages/linux/dconf-write.md

```diff
@@ -0,0 +1,24 @@
+# dconf write
+
+> Write a value to a dconf database path.
+> More information: <https://developer.gnome.org/dconf>.
+
+- Write a string to a dconf path (note the nested quotes):
+
+`dconf write {{/example/dconf/path}} "'{{Example Value}}'"`
+
+- Write a boolean to a dconf path:
+
+`dconf write {{/example/dconf/path}} {{true}}`
```

Comment:
```suggestion
`dconf write {{/example/dconf/path}} {{true_or_false}}`
```

Don't commit this until we have had others opinions

Reply:
If anything it'd be `true|false`, but I've seen booleans represented by specifying both or just one

Reply:
from what I have seen we use snake to separate possible commands, some may not understand what the pipe means for usage

Reply:
Actually I only see `true|false` in `pages/common/qutebrowser.md`

Reply:
Can you link an example? I'm getting nothing from `rg -F 'true_or_false'`

Reply:
hmm, yes. Again we need to wait for others opinion

Reply:
I'd prefer `true|false` here, because it indicates that you can only use those two values, and nothing else.

Reply:
```suggestion
`dconf write {{/example/dconf/path}} {{true|false}}`
```